apps/gatsby/src/Sidebar.spec.js

```diff
@@ -32,43 +45,22 @@ describe('Gatsby App Sidebar', () => {
   afterEach(cleanup);
 
   it('should match snapshot', () => {
+    mockSdk.entry.getSys.mockReturnValue(getMockContent());
     const { container } = render(<Sidebar sdk={mockSdk} />);
 
     expect(container).toMatchSnapshot();
   });
 
-  it('should debounce the fetch', async () => {
+  it('should call onSysChanged and create a manifestId', async () => {
     const mockFetch = jest.fn(() => Promise.resolve());
-    let timerComplete = false;
     global.fetch = mockFetch;
     mockSdk.entry.onSysChanged.mockImplementationOnce(fn => {
-      fn();
-      setTimeout(() => {
-        timerComplete = true;
-      }, 1000);
+      fn({ ...getMockContent() });
       return jest.fn();
     });
 
-    render(<Sidebar sdk={mockSdk} />);
+    mockSdk.entry.getSys.mockReturnValue(getMockContent());
```

Comment:
Do we need to add an assertion here?

Reply:
No, this is just to avoid those uncaught errors we were seeing. I'll try and remove this in favor of a default value in a followon PR!